src/linagora.esn.calendar/app/event/form/open/event-form.service.js

```diff
@@ -48,14 +48,14 @@ require('../../form/event-form.directive.js');
       $modal({
         template: require("../modals/event-form-modal.pug"),
         resolve: {
-          event: function(calEventUtils) {
+          event: function() {
             return calEventUtils.getEditedEvent();
           },
           relatedEvents: function() {
             return relatedEvents;
           }
         },
-        controller: function($scope, event, relatedEvents) {
+        controller: /* @ngInject */ function($scope, event, relatedEvents) {
```

Comment:
This one was f**cking hard to find without any explicit log on angular compile from bootstrap modal